src/config.hpp

```diff
@@ -490,13 +490,27 @@ class config
 	/**
 	 * Get a deprecated attribute without a direct substitute,
 	 * and log a deprecation message
+	 * @deprecated Use the overload that takes a version_info instead.
```

Comment:
This is currently only used in some WFL code that's marked FOR_REMOVAL in 1.17, so this can be removed with that code.